<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourSelection</class>
 <widget class="QWidget" name="ContourSelection">
  <layout class="QVBoxLayout" name="m_main_layout">
   <item>
    <widget class="QLabel" name="m_contour_area_label"/>
   </item>
   <item>
    <widget class="gui::widgets::ScrollArea" name="m_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="m_contour_selection_widget"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_button_layout">
     <item>
      <widget class="QPushButton" name="m_previous_button">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset theme="go-previous"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_next_button">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset theme="go-next"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widgets::ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>gui/widgets/ScrollArea.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
